src/test/kotlin/org/rust/lang/core/completion/RsCompletionTest.kt

```diff
@@ -934,7 +935,28 @@ class RsCompletionTest : RsCompletionTestBase() {
         }
     """)
 
-    @IgnoreInNewResolve
+    @MockEdition(Edition.EDITION_2018)
+    fun `test private reexport of public function`() = checkNoCompletion("""
+        mod mod1 {
+            pub fn foo() {}
+        }
+        mod mod2 {
+            use crate::mod1::foo as bar;
+        }
+
+        fn main() {
+            mod2::b/*caret*/
+        }
+    """)
+
+    // there was error in new resolve when legacy textual macros are always completed
+    @MockEdition(Edition.EDITION_2018)
+    @ProjectDescriptor(WithStdlibRustProjectDescriptor::class)
+    fun `test no completion on empty mod`() = checkNoCompletion("""
```

Comment:
Btw it is possible to write it without std. Can be written in addition to this test.